internal/presentation/snapshots.go

```diff
@@ -73,12 +73,27 @@ type SnapshotMarshaled struct {
 	CaptureTime    string                      `json:"captureTime"`
 	ExpirationTime string                      `json:"expirationTime,omitempty"`
 	Ready          bool                        `json:"ready"`
-	Devices        []storage.Device            `json:"devices"`
```

Comment:
are you changing the storage layer at all? I assume not b/c this is in the presentation layer?